src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -49,6 +49,11 @@ public function __construct(string|array $fileLinkFormat = null, RequestStack $r
         $this->urlFormat = $urlFormat;
     }
 
+    /**
+     * @return string|false
+     *
+     * @psalm-return false|string
```

Comment:
needless

Reply:
Yeah, we should remove all `@psalm-return` added, they were a mistake.